profiler/options.go

```diff
@@ -60,9 +71,10 @@ func (c *config) addProfileType(t ProfileType) {
 }
 
 func defaultConfig() *config {
+	u, _ := urlForSite(defaultSite)
```

Comment:
I'm terrified by errors using `_` and even though this is legit, we never know who's going to change the argument and let the error ignored. This also means that the error is pointless, or that we would have no use for it here. I would suggest instead to restore `defaultAPIURL` and simply use that here. Just like it was before. `defaultSite` is not really needed anyway, right?

Reply:
Sure, I think this makes sense.